clippy.toml

```diff
@@ -1 +0,0 @@
-msrv = "1.60"
```

Comment:
Don't remove this! The goal of this field is that even if you run Clippy on Rust 1.80, Clippy will not suggest things that won't compile on 1.60. (Otherwise, depending on your local Rust version, you'll get different Clippy warnings.)

Reply:
ah, i didn't realize that - thanks! i'll patch it up tomorrow and re-push it then